components/layouts/TheMenu/index.vue

```diff
@@ -192,14 +285,39 @@ export default class TheMenu extends Mixins(MenuMixin) {
     color: var(--link-color);
     margin-right: $spacing-09;
 
-    &:last-child {
+    &_active .bx--list-box__label {
+      color: $purple-70;
+    }
+
+    &_secondary {
+      display: block;
+      padding: $spacing-03 $spacing-05;
       margin-right: 0;
+
+      &:hover {
+        text-decoration: none;
+      }
     }
+  }
 
-    &_active,
-    &_active:hover {
+  // override applink
+  .app-link {
```

Comment:
Instead of using `AppLink` and override its styles, you can use `BasicLink` that is styles-free and add the styles you want